addons/account/security/ir.model.access.csv

```diff
@@ -118,7 +118,7 @@ access_account_payment_term_partner_manager,account.payment.term partner manager
 access_account_payment_term_manager,account.payment.term,model_account_payment_term,account.group_account_manager,1,1,1,1
 access_account_payment_term_line_partner_manager,account.payment.term.line partner manager,model_account_payment_term_line,base.group_user,1,0,0,0
 access_account_payment_term_line_manager,account.payment.term.line,model_account_payment_term_line,account.group_account_manager,1,1,1,1
-access_account_payment_method_readonly,account.payment.method,model_account_payment_method,account.group_account_readonly,1,0,0,0
+access_account_payment_method_readonly,account.payment.method,model_account_payment_method,base.group_user,1,0,0,0
```

Comment:
From the commit message: 
>* Because account.payment.method can be shown on the res.partner form,
  we need to relax the security level to readonly for all users